src/Models/Article.php

```diff
@@ -32,14 +32,11 @@ class Article extends LegoModel implements Sectionable, Indexable, Publishable,
 
     protected $casts = [
         'meta' => 'json',
+        'published_at' => 'date',
```

Comment:
Can we make it a `datetime` instead so we also get the time with it. Not sure that straight up `date` takes time into consideration as well.

```suggestion
        'published_at' => 'datetime',
```